src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -78,10 +83,12 @@ export class SearchMenu extends React.Component {
 
     // end early / clear suggestions if user input is too short
     if (userInput?.length <= 2) {
-      if (this.state.suggestions.length > 0) {
```

Comment:
You could do this instead of the nested if statement if you want:

```jsx
  if (userInput?.length <= 2) {
    this.setState({ suggestions: [], savedSuggestions: [] });
    return;
  }
```

since if `suggestions` and `savedSuggestions` are already empty then React is smart enough to not re-render 🙂

Reply:
Awesome! I didnt know if that would cause some unnecessary state changes or not! good call!